prio/cli/commands.py

```diff
@@ -190,6 +192,8 @@ def verify2(
 
     name = os.path.basename(input_internal)
     outfile = os.path.join(output, name)
+    if not os.path.exists(output):
+        os.makedirs(output)
```

Comment:
I think you could simplify this to just:

`os.makedirs(output, exist_ok=True)`